control-plane/pkg/security/secret_test.go

```diff
@@ -308,6 +308,7 @@ func TestSASLSCRAM256SSL(t *testing.T) {
 	secret := map[string][]byte{
 		"protocol":       []byte("SASL_SSL"),
 		"sasl.mechanism": []byte("SCRAM-SHA-256"),
+		"user.skip":      []byte("true"),
```

Comment:
(Maybe read [this comment](https://github.com/knative-sandbox/eventing-kafka-broker/pull/2341#discussion_r906826776) first)

The test is about SASL + SSL (no mTLS) and is passing a `user.key`.
When configuring SSL (no mTLS) users do not inform that field, but when configuring SSL with mTLS it is needed.

To keep the test covering the same scenario either:

- `user.skip: true` is added.
- `userKey` is removed from the secret.

Maybe my choice of adding `user.skip` is not the best, since we don't expect that field informed, and I should just remove `user.key` from the test. Would that be ok with you?